docs/source/algorithms/Q1DWeighted-v1.rst

```diff
@@ -69,6 +97,14 @@ The wedges output group will have two workspaces: one for the red region, one fo
   :align: center
   :width: 600
 
+In the case of drawn sectors, when doing symmetric integration, symmetric shapes will be grouped together.
+Taking the above example, the shape table will have 4 shapes in it, but the output will only have 2 workspaces, because the
+red shapes and the blue shapes will be grouped.
+If no corresponding symmetric is found for a shape, the algorithm will nonetheless integrate on the projected symmetric too,
```

Comment:
This is only if AsymmWedges is false, right? Perhaps make it clearer that Symmetric/Asymmetric is controlled with that flag before branching into these 2 sections.